aspnetcore/blazor/get-started.md

```diff
@@ -5,163 +5,114 @@ description: Get started with Blazor by building a Blazor app with the tooling o
+1. For a Blazor WebAssembly experience, choose the **Blazor WebAssembly App** template. For a Blazor Server experience, choose the **Blazor Server App** template. Select **Create**.
+
+   For information on the two Blazor hosting models, *Blazor WebAssembly* and *Blazor Server*, see <xref:blazor/hosting-models>.
 
 1. Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> to run the app.
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
 1. Install the [.NET Core 3.1 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.1).
 
-1. Optionally install the [Blazor WebAssembly](xref:blazor/hosting-models#blazor-webassembly) preview template by running the following command:
-
-   ```dotnetcli
-   dotnet new -i Microsoft.AspNetCore.Components.WebAssembly.Templates::3.2.0-rc1.20223.4
-   ```
-   
-   For information on the two Blazor hosting models, *Blazor WebAssembly* and *Blazor Server*, see <xref:blazor/hosting-models>.
-
-   > [!NOTE]
-   > The [.NET Core SDK version 3.1.201 or later](https://dotnet.microsoft.com/download/dotnet-core/3.1) is **required** to use the 3.2 Preview Blazor WebAssembly template. Confirm the installed .NET Core SDK version by running `dotnet --version` in a command shell.
-
 1. Install [Visual Studio Code](https://code.visualstudio.com/).
 
 1. Install the latest [C# for Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) and the [JavaScript Debugger (Nightly)](https://marketplace.visualstudio.com/items?itemName=ms-vscode.js-debug-nightly) extension with `debug.javascript.usePreview` set to `true`.
 
-1. For a Blazor Server experience, execute the following command in a command shell:
+1. For a Blazor WebAssembly experience, execute the following command in a command shell:
 
    ```dotnetcli
-   dotnet new blazorserver -o WebApplication1
+   dotnet new blazorwasm -o WebApplication1
    ```
 
-   For a Blazor WebAssembly experience, execute the following command in a command shell:
+   For a Blazor Server experience, execute the following command in a command shell:
 
    ```dotnetcli
-   dotnet new blazorwasm -o WebApplication1
+   dotnet new blazorserver -o WebApplication1
    ```
 
+   For information on the two Blazor hosting models, *Blazor WebAssembly* and *Blazor Server*, see <xref:blazor/hosting-models>.
+
 1. Open the *WebApplication1* folder in Visual Studio Code.
 
 1. The IDE requests that you add assets to build and debug the project. Select **Yes**.
 
-1. With Blazor Server, run the app using the Visual Studio Code debugger.
+1. For a Blazor WebAssembly app, start the app using the **.NET Core Launch (Blazor Standalone)** launch configuration and then start the browser using the **.NET Core Debug Blazor Web Assembly in Chrome** launch configuration (requires Chrome). For more information, see <xref:blazor/debug#visual-studio-code>.
```

Comment:
Since at this point we don't expect the user to actually debug and we will be updating the VSCode experience to make it a single step process really soon, let's just tell folks to hit F5 to run the app.